internal/limitation/file/internal-autoit-file-limitation.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: (internal) autoit file limitation
+    # capa will detect dozens of capabilities for AutoIt samples,
+    # but these are due to the AutoIt runtime, not the payload script.
+    # so, don't confuse the user with FP matches - bail instead
+    description: |
+      This sample appears to be compiled with AutoIt.
+      
+      AutoIt is a freeware BASIC-like scripting language designed for automating the Windows GUI.
+      capa cannot handle AutoIt scripts. This means that the results will be misleading or incomplete.
+      You may have to analyze the file manually, using a tool like the AutoIt decompiler MyAut2Exe.
+    namespace: internal/limitation/file
+    author: william.ballenthin@fireeye.com
+    scope: file
+    examples:
+      - 55D77AB16377A8A314982F723FCC6FAE
+  features:
+    - or:
+      - match: compiler/autoit
```

Comment:
linter will require a new line at the end

Reply:
linter bug #601

Reply:
fixed in https://github.com/fireeye/capa/commit/2706a7171e7dcd7dec689229d2ebe1bf4e735bbb